packages/create-snap-cli/src/cmds/init/index.ts

```diff
@@ -3,18 +3,20 @@ import yargs from 'yargs';
 
 import builders from '../../builders';
 import { YargsArgs } from '../../types/yargs';
-import { build } from '../build/buildHandler';
 import { initHandler } from './initHandler';
 
-export = {
-  command: ['init [directory]', 'i [directory]'],
-  desc: 'Initialize Snap package',
+const initCommand = {
+  command: ['$0 [directory]'],
+  desc: 'Initialize MetaMask Snaps project',
   builder: (yarg: yargs.Argv) => {
     yarg.positional('directory', builders.directory);
+    return yarg;
   },
-  handler: async (argv: YargsArgs) => init(argv),
+  handler: init,
 };
 
+export default initCommand;
```

Comment:
You could just export `initCommand` as named export. That simplifies the `index.ts` too.